p/themes/Ansum/ansum.css

```diff
@@ -417,14 +417,14 @@ th {
 	color: #ca7227;
 	transition: all 0.15s ease-in-out;
 }
-#bigMarkAsRead:hover {
+#bigMarkAsRead.big:hover {
```

Comment:
@math-GH Please double-check

Reply:
oh, wait. The ":hover" was commited via another PR

Reply:
see #7490

Reply:
Thanks :-)